bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-messages/component.jsx

```diff
@@ -83,20 +85,44 @@ const intlMessages = defineMessages({
 });
 
 class UserMessages extends Component {
+  constructor() {
+    super();
+
+    this.getMessagesListRef = this.getMessagesListRef.bind(this);
+    this.getMessageItemsRef = this.getMessageItemsRef.bind(this);
+  }
+
   componentDidMount() {
     if (!this.props.compact) {
       this._msgsList.addEventListener(
         'keydown',
         event => this.props.rovingIndex(
           event,
-          this._msgsList,
-          this._msgItems,
+          this.getMessagesListRef(),
+          this.getMessageItemsRef(),
           this.props.openChats.length,
         ),
       );
+
+      document.addEventListener(
+        'click',
+        event => this.props.resetIndex(
```

Comment:
This is another thing that should have been done via props/state.
You are using findDOMNode to find and manipulate elements directly which is not recommended and might potentially fail later (we've been warned in React 16 release), to modify the attributes directly.

But in fact what should happen - you update state and these parts receive different tabIndex via props or state.